### PR TITLE
Merge HugeGraph-Hubble-fe v1.2.0

### PR DESCRIPTION
**Pull request by OshotOkill**

### CODE REVIEW

File: hubble-fe/src/stores/utils/index.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export function checkIfLocalNetworkOffline(error: any) {
+  if (!error.response) {
+    throw new Error('网络错误，请查看您的本地连接');
```
Comment (javeme) [inline hubble-fe/src/stores/utils/index.ts]: TODO: use i18n instead

File: hubble-fe/src/stores/GraphManagementStore/metadataConfigsStore/edgeTypeStore.ts [hunk 2]
```diff
@@ -0,0 +1,1403 @@
+import { observable, action, flow, computed } from 'mobx';
```
Comment (javeme) [inline hubble-fe/src/stores/GraphManagementStore/metadataConfigsStore/edgeTypeStore.ts]: metadataConfigsStore/edgeTypeStore.ts 
prefer to rename metadata to schema

File: hubble-fe/src/stores/GraphManagementStore/dataAnalyzeStore.ts [hunk 3]
```diff
@@ -0,0 +1,1236 @@
+import { createContext } from 'react';
```
Comment (javeme) [inline hubble-fe/src/stores/GraphManagementStore/dataAnalyzeStore.ts]: dataAnalyzeStore.ts -- Keep a consistent file naming style, such as capitalizing the first letter.

File: hubble-fe/src/stores/utils/index.ts (hunk 1 above)
Comment (OshotOkill) [reply hubble-fe/src/stores/utils/index.ts]: No i18n support before v1.2.1 here, plan to add it in v1.3.0

File: hubble-fe/src/stores/GraphManagementStore/dataAnalyzeStore.ts (hunk 3 above)
Comment (OshotOkill) [reply hubble-fe/src/stores/GraphManagementStore/dataAnalyzeStore.ts]: It's the exact api name which `React` exports

File: hubble-fe/src/stores/GraphManagementStore/metadataConfigsStore/edgeTypeStore.ts (hunk 2 above)
Comment (OshotOkill) [reply hubble-fe/src/stores/GraphManagementStore/metadataConfigsStore/edgeTypeStore.ts]: The name `metadata` is heavily used in `hubble-fe` (about 800+ places), rename them all could be fragile and it's not that necessary

**Pull request merged**